deploy/helm/kubecf/values.yaml

```diff
@@ -138,6 +144,9 @@ features:
   # add autoscaler
   autoscaler:
     enabled: false
+  # add sle15 stack
```

Comment:
Could you remove this comment? It's not adding anything useful to the context, nor have we a complex logic here that needs explanation.

We also want to get rid of the autoscaler one.